vendor/github.com/openshift/api/install.go

```diff
@@ -51,4 +82,41 @@ var (
 	)
 	// Install is a function which adds every version of every openshift group to a scheme
 	Install = schemeBuilder.AddToScheme
+
+
+	kubeSchemeBuilder = runtime.NewSchemeBuilder(
+		kadmissionv1beta1.AddToScheme,
+		kadmissionregistrationv1alpha1.AddToScheme,
+		kadmissionregistrationv1beta1.AddToScheme,
+		kappsv1.AddToScheme,
+		kappsv1beta1.AddToScheme,
+		kappsv1beta2.AddToScheme,
+		kauthenticationv1.AddToScheme,
+		kauthenticationv1beta1.AddToScheme,
+		kauthorizationv1.AddToScheme,
+		kauthorizationv1beta1.AddToScheme,
+		kautoscalingv1.AddToScheme,
+		kautoscalingv2beta1.AddToScheme,
+		kbatchv1.AddToScheme,
+		kbatchv1beta1.AddToScheme,
+		kbatchv2alpha1.AddToScheme,
```

Comment:
Do we want to enable this alpha, I'm thinking about deprecating it in k8s asap and get rid of it.